@here/harp-mapview/test/VisibleTileSetTest.ts

```diff
@@ -268,6 +268,46 @@ describe("VisibleTileSet", function() {
         assert.equal(renderedTiles.size, 0);
     });
 
+    it("dependencies of tiles are added to visible tile set", function() {
+        const tileKey1 = TileKey.fromMortonCode(371506850);
+        const tileKey2 = TileKey.fromMortonCode(371506851);
+
+        // Test where we have 2 actually visible tiles (in the sense that they are in the frustum,
+        // see visibleTileKeys below) and one visible tile because it is a dependency.
+        // tslint:disable-next-line: no-string-literal
+        fixture.vts["getVisibleTileKeysForDataSources"] = sinon.stub().returns({
+            tileKeys: [
+                {
+                    dataSource: fixture.ds[0],
+                    visibleTileKeys: [new TileKeyEntry(tileKey1, 0), new TileKeyEntry(tileKey2, 0)]
+                }
+            ],
+            allBoundingBoxesFinal: true
+        });
+
+        // Adding the dependency to make sure it is visible
+        const tile1 = fixture.vts.getTile(fixture.ds[0], tileKey1, 0, 0);
+        assert.notEqual(tile1, undefined);
+        const tileKey3 = TileKey.fromMortonCode(371506852);
+        tile1!.dependencies.push(tileKey3);
```

Comment:
this is where the dependency is added from another tile to tile 1, i.e. when tile 1 is visible, tile with tileKey 3 will also be shown